stablehlo/reference/Ops.cpp

```diff
@@ -1144,15 +1165,19 @@ SmallVector<Tensor> evalReduceOp(ArrayRef<Tensor> inputs,
       resultIndex.push_back(inputIndexElement);
     }
 
-    SmallVector<Tensor> bodyArgs;
-    for (auto [result, initValue] : llvm::zip(results, initValues))
-      bodyArgs.push_back(Tensor(initValue.getType(), result.get(resultIndex)));
-    for (auto [input, initValue] : llvm::zip(inputs, initValues))
-      bodyArgs.push_back(Tensor(initValue.getType(), input.get(*inputIt)));
+    SmallVector<InterpreterValue> bodyArgs;
+    for (auto [result, initValue] : llvm::zip(results, initValues)) {
+      Tensor tensor(initValue.getType(), result.get(resultIndex));
+      bodyArgs.emplace_back(tensor);
+    }
+    for (auto [input, initValue] : llvm::zip(inputs, initValues)) {
+      Tensor tensor(initValue.getType(), input.get(*inputIt));
+      bodyArgs.emplace_back(tensor);
+    }
```

Comment:
Can we keep the original-style loop bodies without temporary variables and curly braces? Same question for evalScatterOp.